### PR TITLE
Moved GRAVITY_* constants to Item model class

### PR DESCRIPTION
**Pull request by Quix0r**
According to @MrPetovan constants in `boot.php` that are not used in configuration files can be safely moved to proper classes. This PR moves all `GRAVITY_*` constants to `Friendica\Model\Item` class.

### CODE REVIEW

File: mod/oexchange.php [hunk 1]
```diff
@@ -109,7 +109,7 @@ function oexchange_content(App $a) {
 	$description = !empty($_REQUEST['description']) ? trim($_REQUEST['description']) : '';
 	$tags        = !empty($_REQUEST['tags'])        ? trim($_REQUEST['tags'])        : '';
 
-	$s = \Friendica\Content\Text\BBCode::embedURL($url, true, $title, $description, $tags);
+	$s = BBCode::embedURL($url, true, $title, $description, $tags);
```
Comment (MrPetovan) [inline mod/oexchange.php:114]: The related `use Friendica\Content\Text\BBCode;` statement seems to be missing.

File: mod/oexchange.php [hunk 2]
```diff
@@ -119,9 +119,9 @@ function oexchange_content(App $a) {
 
 	$post['profile_uid'] = local_user();
 	$post['return'] = '/oexchange/done';
-	$post['body'] = Friendica\Content\Text\HTML::toBBCode($s);
```
Comment (MrPetovan) [inline mod/oexchange.php:122]: The related `use Friendica\Content\Text\HTML;` statement seems to be missing.

File: mod/tagger.php [hunk 3]
```diff
@@ -116,41 +116,41 @@ function tagger_content(App $a) {
...
+		'gravity'       => Item::GRAVITY_COMMENT,
+		'parent'        => $item['id'],
+		'thr-parent'    => $item['uri'],
+		'owner-name'    => $item['author-name'],
+		'owner-link'    => $item['author-link'],
+		'owner-avatar'  => $item['author-avatar'],
+		'author-name'   => $contact['name'],
+		'author-link'   => $contact['url'],
+		'author-avatar' => $contact['thumb'],
+		'body'          => sprintf( $bodyverb, $ulink, $alink, $plink, $termlink),
```
Comment (MrPetovan) [inline mod/tagger.php]: Code standards.
```suggestion
		'body'          => sprintf($bodyverb, $ulink, $alink, $plink, $termlink),
```

File: view/theme/frio/theme.php [hunk 4]
```diff
@@ -33,7 +33,9 @@
 use Friendica\Database\DBA;
 use Friendica\DI;
 use Friendica\Model;
```
Comment (MrPetovan) [inline view/theme/frio/theme.php:35]: Is this still needed?

**Review by MrPetovan — CHANGES_REQUESTED** [review]

**Comment by annando** [comment]
Overall the usage of an IDE with built-in syntax checker would help here. Especially with this huge amount of files, the possibility of syntax errors is immense.

It would be much better, not to have performed multiple steps (exchanging the constants and code reformatting) in one huge PR. Also it would had been better to keep the old constants for some time and just deprecating them, since they are also used in the addons.

File: view/theme/frio/theme.php (hunk 4 above)
Comment (Quix0r) [reply view/theme/frio/theme.php:35]: I don't know. But that the fact that themes can have access to database (circumventing the controller) is scary.

**Comment by Quix0r** [comment]
> Overall the usage of an IDE with built-in syntax checker would help here. Especially with this huge amount of files, the possibility of syntax errors is immense.

I strongly disagree here. Better more unit-testing which would "on-the-fly" test this for us as a by-product.

**Comment by Quix0r** [comment]
And for addons, I have also made a branch ready for PR.

**Review by MrPetovan — APPROVED** [review]

**Pull request merged**